server/src/api/commits.ts

```diff
@@ -19,6 +19,7 @@
  */
 
 import express, {Request, Response, NextFunction} from 'express';
+import _ from 'lodash';
```

Comment:
```suggestion
import isEqual from 'lodash/isEqual';
```
Prefer this because let's not import the entire library when we only need 1 function! Also, not a fan of `_` because it is a commonly used symbol to refer to an ignored variable (although I know lodash like to refer to themselves as `_` haha). If you like, you can also name it `_isEqual` if you want to be clear the func is frm lodash.
On another hand, is lodash really necessary?

Reply:
@hzxcaryn As discussed over Chat, will standardise the check with this https://github.com/googleinterns/gpay-group-buy/pull/195/files#diff-5babba68806a5317f63daf9e8dfbd212

Reply:
@hzxcaryn We no longer have this check after changing the endpoint to Complete Commit instead of Edit Commit.